application/src/main/kotlin/application/GraphCommand.kt

```diff
@@ -1,41 +1,60 @@
 package application
 
-import `in`.specmatic.core.SpecmaticConfigJson
 import `in`.specmatic.core.azure.AzureAPI
 import `in`.specmatic.core.azure.PersonalAccessToken
 import `in`.specmatic.core.git.getPersonalAccessToken
+import `in`.specmatic.core.loadSpecmaticJsonConfig
 import `in`.specmatic.core.log.CompositePrinter
 import `in`.specmatic.core.log.Verbose
 import `in`.specmatic.core.log.logger
 import `in`.specmatic.core.pattern.ContractException
 import `in`.specmatic.core.utilities.exitWithMessage
 import picocli.CommandLine
 import picocli.CommandLine.Option
-import picocli.CommandLine.Parameters
 import java.util.concurrent.Callable
 
 @CommandLine.Command(name = "graph",
     mixinStandardHelpOptions = true,
     description = ["Dependency graph"])
 class GraphCommand: Callable<Unit> {
-    @CommandLine.Command(name = "consumers", description = ["Display a list of services depending on contracts in this repo"])
-    fun consumers(@Option(names = ["--verbose"], description = ["Print verbose logs"]) verbose: Boolean = false, @Option(names = ["--azureBaseURL"], description = ["Azure base URL"], required = true) azureBaseURL: String) {
-        if(verbose)
+    @CommandLine.Command(
+        name = "consumers",
+        description = ["Display a list of services depending on contracts in this repo"]
+    )
+    fun consumers(
+        @Option(names = ["--verbose"], description = ["Print verbose logs"]) verbose: Boolean = false,
+        @Option(
+            names = ["--azureBaseURL"],
+            description = ["Azure base URL"],
+            required = true
+        ) azureBaseURL: String
+    ) {
+        if (verbose)
             logger = Verbose(CompositePrinter())
 
-        val configJson = SpecmaticConfigJson.load()
+        val configJson = loadSpecmaticJsonConfig(null)
```

Comment:
are we passing the null intentionally?

Reply:
Yeah right now the method expects the 'configFileName' parameter. I'll fix it so that we don't have to pass the null explicitly.